gateway/gateway-controller/pkg/config/xds_tls_test.go

```
@@ -0,0 +1,364 @@
+	require.NoError(t, err)
+
+	clientCert, err := tls.LoadX509KeyPair(clientCertPath, clientKeyPath)
+	require.NoError(t, err)
+	clientCAPool := x509.NewCertPool()
+	clientCAPool.AddCert(caCert)
+
+	dial := func(t *testing.T, clientCurves []tls.CurveID) tls.CurveID {
+		t.Helper()
+		ln, err := tls.Listen("tcp", "127.0.0.1:0", serverTLSConfig)
+		require.NoError(t, err)
+		defer ln.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `ln.Close` is not checked

(errcheck)